tests/readPreference/readpreference-ctor_error-005.phpt

```diff
@@ -1,18 +0,0 @@
---TEST--
-MongoDB\Driver\ReadPreference construction (invalid string mode)
```

Comment:
This test was redundant as `readpreference-ctor_error-001.phpt` uses an invalid string value.